### PR TITLE
completed status component

### PR DESCRIPTION
**Pull request by mikexd-dev**
# Pull Request Overview

### Description:

> _Write a short description for this PR_

Status component has 2 main states, upcoming vs launched, displaying mint date vs displaying minted amount/totalsupply

### Key Changes:

> _List out key changes made in this PR_

https://menyala.atlassian.net/browse/AUTH-44

<br>

---

### Link to story:

< Write your response here >

### CODE REVIEW

File: lib/checkPremint.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import moment from "moment";
```
Comment (terrytjw) [inline lib/checkPremint.ts:1]: are we using both `moment` and `date-fns`?

File: components/Status.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+const Status = ({ mintDate, mintedAmount, totalSupply }: StatusProps) => {
+  const isPremintCollection = isPremint(mintDate);
+  let statusText = isPremintCollection ? "UPCOMING" : "LAUNCHED";
+  let metadata = isPremintCollection
+    ? `Starts at ${format(new Date(mintDate), "dd MMMM yyyy")}`
+    : `Total minted (${mintedAmount}/${totalSupply})`;
+  let bgColor = isPremintCollection ? "auth-dark-blue" : "auth-blue";
+  let progressColor = isPremintCollection ? "progress-accent" : "white";
+
+  console.log(progressColor, isPremintCollection);
```
Comment (terrytjw) [inline components/Status.tsx:20]: remove log statement

**Review by terrytjw — CHANGES_REQUESTED** [review]
some questions and changes

File: lib/checkPremint.ts (hunk 1 above)
Comment (mikexd-dev) [reply lib/checkPremint.ts]: removed momentjs

**Review by terrytjw — APPROVED** [review]

**Pull request merged**